bin/ar

```diff
@@ -225,7 +225,7 @@ sub extractMember {
 				   oct($attr->[4]), int($attr->[1]));
     }
 
-    my $out = new FileHandle ">$name" or die "$name: $!\n";
+    my $out = FileHandle->new ">$name" or die "$name: $!\n";
```

Comment:
Let's put some parens around the argument list (for this line and all the others like it)

FileHandle->new( ">$name" )

Reply:
Added the parens